### ISSUE TITLE
Permissions Branch: New user unable to view templates or create board

### ISSUE THREAD

**Issue by chenilim** [mattermost/focalboard#2552]
## Steps to reproduce the behavior

1. Create a new user
2. Open Boards
3. Select a template and try to create a board
4. Nothing happens (errors in browser console), and no template preview

## Expected behavior

Able to view template preview and create board

## Screenshots (optional)

<img width="1124" alt="Screen Shot 2022-03-16 at 5 15 52 PM" src="https://user-images.githubusercontent.com/46905241/158712313-8606ea6a-df60-478e-8796-6c07d984530e.png">

## Edition and Platform

 - Edition: Mattermost Boards (plugin)
 - Version: permissions-feature-branch
 - Browser and OS: Chrome on Mac

## Additional context (optional)

Able to view template and create board with SysAdmin account, but not a newly created one.

Console errors:
```
octoClient.ts:269          GET https://boards-permissions.test.mattermost.cloud/plugins/focalboard/api/v1/boards/btr5nxxr8u38x9guhq87f98pafr/blocks?all=true 403 (Forbidden)
octoClient.ts:636          POST https://boards-permissions.test.mattermost.cloud/plugins/focalboard/api/v1/boards/btr5nxxr8u38x9guhq87f98pafr/duplicate?asTemplate=false&toTeam=3n3k5fb7tibsipz5k8aqjpd1fw 403 (Forbidden)
```

**Comment by wuwinson** [comment]
May get addressed by #2370.

**Comment by sbishel** [comment]
Closing in favor of - https://github.com/mattermost/focalboard/issues/2712
